include/msgpack/adaptor/cpp11/unordered_map.hpp

```diff
@@ -42,9 +42,9 @@ struct convert<std::unordered_map<K, V>> {
         for(; p != pend; ++p) {
             K key;
             p->key.convert(key);
-            p->val.convert(tmp[key]);
+            p->val.convert(tmp[std::move(key)]);
         }
-        tmp.swap(v);
+        v = std::move(v);
```

Comment:
Whoops, sorry. I only discovered today that I had previously forgotten this occurrence, and apparently exercised less caution on this one. Uploaded an updated commit, the tests are now passing locally for me.